modules/programs/alacritty.nix

```diff
@@ -18,15 +21,15 @@ in {
       };
 
       settings = mkOption {
-        type = yamlFormat.type;
+        type = configFormat.type;
```

Comment:
Hmm, unfortunately cannot really do this either due to the problem with manual depending on configuration. I wonder if it may be best to simply switch unconditionally to TOML, since that seems to be the future format. Then add an assertion that fails the build if the version is below 0.13.

Adding something like

``` nix
assertions = [
  {
    assertion = lib.versionAtLeast cfg.package.version "0.13";
    message = "The Alacritty module requires Alacritty version 0.13 or later.";
  }
];
```

to the `config` attribute might do the trick.

Reply:
Yeah, that would simplify things a lot, but I didn't want to break backwards compat. Also then we have an issue with the PR for the 0.13 update in nixpkgs and this PR needing to be merged at the same time.

Reply:
The types for toml and yaml are currently the same though, so we can just use the toml type and remove the yaml support in the near future once the PR in nixpkgs was merged.

Reply:
Changed to use the toml formatter unconditionally for the type, it's the same as the yaml type anyway.

Reply:
That sounds like a good solution. Thanks!